tools/build-toolchain.sh

```diff
@@ -16,7 +16,7 @@ set -euo pipefail
 IFS=$'\n\t'
 
 # Set N64_INST before calling the script to change the default installation directory path
-INSTALL_PATH="${N64_INST:-/usr/local}"
+INSTALL_PATH="${N64_INST}"
```

Comment:
Should we really change the default? There might be setups (potentially automated) depending on the default location. Also what happens if `N64_INST` does not exist in the current environment with this setup?

Reply:
> Should we really change the default? There might be setups (potentially automated) depending on the default location.

I don't think there are many CIs that are building the toolchain via this script. The point is that everything you do with libdragon *after* building the toolchain requires this variable to be set. So maybe it is better to ask the user to set it before building the toolchain, so they don't get confused. Also, this gives explicit visibility to the fact that you can change the installation directory (and it has to be declared before building -- you cannot change it after building).

> what happens if N64_INST does not exist in the current environment with this setup?

The script aborts with an error because we are in strict mode:

Reply:
Looks good!

Reply:
I forgot Make's rules, will this error if N64_INST is not set? If not could we do something like
```Makefile
ifndef N64_INST
$(error Please set the environment variable N64_INST to the location to install the tool chain. Check README.md for more details)
endif
```

Reply:
We could do that but it would have to be duplicated in all Makefiles of all libdragon projects. In general I would say the Makefile is under the project responsibility, so we’re not helping in solving the problem to new users, which might hit the problem while trying to compile eg Flappy Bird